html-skeleton/html-skeleton-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# html-skeleton-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Where do you put non-visible content about the HTML document?
+You can use the ccomment tags, <!-- and -->, to put content in the html document that will not be visible on to the viewer of the webpage.
```

Comment:
Yes, but it's more-so asking what specific elements do you put non-visible content between? The `body` tag, the `head` tag, etc.